apps/www/src/registry/default/plate-ui/link-floating-toolbar.tsx

```diff
@@ -67,26 +74,24 @@ export function LinkFloatingToolbar({ state }: LinkFloatingToolbarProps) {
     editButtonProps,
     unlinkButtonProps,
   } = useFloatingLinkEdit(editState);
+  const inputProps = useFormInputProps({
+    preventDefaultOnEnterKeydown,
+    onKeyDownCaptureCallback,
```

Comment:
can be removed

Reply:
Again, not sure I fully understand the amend here, sorry. By removing this there would be no way of adding the `onKeyDownCapture` callback to the main wrapping element of the `<LinkFloatingToolbar />` as it does not allow for any props other than `state` so, surely, the user would be required to wrap `<LinkFloatingToolbar />` with a cosmetic `<div>` on which they could then add the output of the `useFormInputProps` to.

The amend above which says to add `preventDefaultOnEnterKeydown: true,` on line 78 seems to contradict this also. Sorry mate, just not sure on what is requested/why it is requested.